openapi/covid19/types.bal

```diff
@@ -14,6 +14,95 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/http;
+
+@display {label: "Connection Config"}
+public type ConnectionConfig record {|
```

Comment:
```suggestion
# Provides a set of configurations for controlling the behaviours when communicating with a remote HTTP endpoint.
@display {label: "Connection Config"}
public type ConnectionConfig record {|
```

Reply:
+1

Reply:
How does this label show up in Choreo UI @SkNuwanTissera ? 
Let's say I want to use two Open API connectors in same application. Then in UI, both configs related to the two connectors will be shown as  `Connection Config` right? What abt generating display label as `<conn name> Connection Config` ? @indikasampath2000 @aneeshafedo

Reply:
There were no use of `@display` when rendering in the UIs. Our handwritten connectors also having the `@display` label similar to this.